src/sass/main.scss

```diff
@@ -206,3 +206,38 @@ body {
 	&--android {
 	}
 }
+
+
+// footer
+
+.foot-container{
+	margin-top: 4rem;
+
+	& .logo{
```

Comment:
don't need the & here as you are just nesting.
& is needed if you want to add something at the end of the parent tag for example.

Reply:
alright noted